help-docs/FAQ/collab-token.md

```diff
@@ -0,0 +1,51 @@
+# Get $COLLAB Tokens
+
+If you have recently created a new wallet address and are ready to acquire $COLLAB tokens, follow these steps to successfully add $COLLAB tokens to your new wallet:

```

Comment:
we should mention both personas, new and old wallet address.

Possibilities are:
1. Completely new wallet without any assets(Fund wallet with Optimism ETH from a existing wallet or centralised exchange and then swap the Optimism ETH with COLLAB tokens)
2. Old wallet but only has Mainnet eth(need to swap with Optimism ETH and then swap the Optimism ETH with COLLAB tokens)
3. Old wallet and has Optimism ETH(just swap with COLLAB tokens)